docs/customization_guide/inference_protocols.md

```diff
@@ -476,6 +476,13 @@ JavaCPP-presets. You can do this using the following steps:
 
 1. Create the JNI binaries in your local repository (`/root/.m2/repository`)
    with [`javacpp-presets/tritonserver`](https://github.com/bytedeco/javacpp-presets/tree/master/tritonserver)
+   For C-API Wrapper Java bindings, you need to install some build specific dependencies like cmake. You can either do this with:
+   ```bash
+    $ git clone https://github.com/triton-inference-server/developer_tools.git
+    $ bash -x ../install_test_dependencies_and_build.sh
```

Comment:
Should this be `install_test_dependencies_and_build.sh` or `install_dependencies_and_build.sh` ?

Reply:
good point updated https://github.com/triton-inference-server/developer_tools/pull/29